src/xdp-utils.c

```diff
@@ -2187,6 +2187,28 @@ xdp_app_info_map_pids (XdpAppInfo  *app_info,
   return ok;
 }
 
+
+gboolean
+xdg_app_info_map_tids (XdpAppInfo  *app_info,
+                       pid_t        owner_pid,
+                       pid_t       *tids,
+                       guint        n_tids,
+                       GError     **error)
+{
+  char proc_dir[31] = {0, };
+  snprintf (proc_dir, sizeof(proc_dir), "/proc/%u/task", (guint) owner_pid);
```

Comment:
```suggestion
  g_autofree char *proc_dir = g_strdup_printf ("/proc/%d/task", owner_pid);
```

Was there a reason you used %u, at a glance its an int?

Reply:
Because I did a lot of copy/paste assuming the code was right, like this https://github.com/flatpak/xdg-desktop-portal/blob/main/src/xdp-utils.c#L1776
(it's not the only place)

Reply:
It's probably not a problem. I don't think negative pids exist.

Reply:
changed to use `g_strdup_printf()`